### PR TITLE
Remove unnecessary information dialog, throw error when we cannot find swift

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: src/toolchain/toolchain.ts [hunk 1]
```diff
@@ -119,14 +119,13 @@ export class SwiftToolchain {
                 // use `type swiftc` to find `swiftc`. Run inside /bin/sh to ensure
                 // we get consistent output as different shells output a different
                 // format. Tried running with `-p` but that is not available in /bin/sh
-                const { stdout } = await execFile("/bin/sh", ["-c", "type swiftc"]);
-                const swiftcMatch = /^swiftc is (.*)$/.exec(stdout);
-                if (swiftcMatch) {
-                    const swiftc = swiftcMatch[1];
-                    vscode.window.showInformationMessage("swiftc: ${swiftc}");
-                    return path.dirname(path.dirname(path.dirname(swiftc)));
+                const { stdout } = await execFile("/bin/sh", ["-c", "type swift"]);
```
Comment (stevapple) [inline src/toolchain/toolchain.ts:122]: There is actually another situation: `/usr/bin/swift` can be a symlink into `/path/to/toolchain/usr/bin/swift-frontend`. Everything can function well, but the toolchain path is wrong.

File: src/toolchain/toolchain.ts [hunk 2]
```diff
@@ -119,14 +119,13 @@ export class SwiftToolchain {
                 // use `type swiftc` to find `swiftc`. Run inside /bin/sh to ensure
                 // we get consistent output as different shells output a different
                 // format. Tried running with `-p` but that is not available in /bin/sh
```
Comment (stevapple) [inline src/toolchain/toolchain.ts:121]:
```suggestion
                // use `type swift` to find `swift`. Run inside /bin/sh to ensure
                // we get consistent output as different shells output a different
                // format. Tried running with `-p` but that is not available in /bin/sh
```

**Review by stevapple — CHANGES_REQUESTED** [review]
Also needs to handle `swift` missing on Windows and macOS.

File: src/toolchain/toolchain.ts (hunk 1 above)
Comment (adam-fowler) [reply src/toolchain/toolchain.ts:122]: What situation is that?

Comment (adam-fowler) [reply src/toolchain/toolchain.ts:122]: anyway fix is in

**Comment by adam-fowler** [comment]
@swift-server-bot test this please

**Review by 0xTim — APPROVED** [review]

**Pull request merged**